src/main/java/org/broadinstitute/hellbender/utils/read/markduplicates/ReadEnds.java

```diff
@@ -36,16 +36,16 @@
     public void setTile(final short tile) { this.tile = tile; }
 
     @Override
-    public short getX() { return this.x; }
+    public int getX() { return this.x; }
 
     @Override
-    public void setX(final short x) { this.x = x; }
+    public void setX(final int x) { this.x = (short)x; }
 
     @Override
-    public short getY() { return this.y; }
+    public int getY() { return this.y; }
 
     @Override
-    public void setY(final short y) { this.y = y; }
+    public void setY(final int y) { this.y = (short)y; }
```

Comment:
We should add checks here that it's not > maxshort.

Reply:
done